cmd/cmd.go

```diff
@@ -78,6 +78,20 @@ func addDatabase(c *cobra.Command) *cobra.Command {
 	return c
 }
 
+var extraIndexesCmd = &cobra.Command{
+	Use:   "extra-indexes index-1 [index-2 …]",
+	Short: "Creates extra indexes in the JSON fields",
+	Args:  cobra.MinimumNArgs(1),
+	RunE: func(_ *cobra.Command, args []string) error {
```

Comment:
`args` é genérico demais. Que tal?

```suggestion
	RunE: func(_ *cobra.Command, idxs []string) error {
```